src/scripts/content/todoist.js

```diff
@@ -144,3 +219,14 @@ function getProjectNames (elem) {
     return projectNames;
   };
 }
+
+function isParentProject (elem) {
```

Comment:
Maybe call the function `getParentProject`?

Reply:
I was debating this one - I think `getParentProject` is a bit misleading because it doesn't necessary get the parent project - it checks if the parent _is_ a project and returns that only if it's true. So it might return nothing. Happy to change though if you think it's more suitable :+1:

Reply:
Quite valid, but the `isX` pattern implies a `boolean` to me.
Maybe `getParentIfPresent` or something like that?

Reply:
Good idea, changing to that :pray:

Reply:
Seems good 👍